ref-exchange/src/lib.rs

```diff
@@ -529,7 +541,7 @@ mod tests {
         // Exchange fees left in the pool as liquidity + 1m from transfer.
         assert_eq!(
             contract.get_pool_total_shares(0).0,
-            33337501041992301475 + 1_000_000
+            100012503125976904 + 1_000_000
```

Comment:
may be it's better to change the exchange and referral fee in `setup_contract` to 1600 and 400 so these numbers continue matching?

Reply:
good advice, change them in upgrade func maybe better.